<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>531</width>
    <height>476</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>27</width>
    <height>24</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="companyType">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Select a company type</string>
    </property>
    <widget class="QRadioButton" name="profitradioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>For profit</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="nonProfitRadioButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Not for profit</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="companyData">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>211</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Enter company data</string>
    </property>
    <widget class="QLabel" name="nameLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="dateLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>50</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Date</string>
     </property>
    </widget>
    <widget class="QLabel" name="employeesLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Employees</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="charitableBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>5</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <family>Yu Gothic Light</family>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Charitable?</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="namelineEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>111</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string extracomment="Use CamelCase typing"/>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>50</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="employeesSpinBox">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QTextEdit" name="profitDisplayBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>251</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="npDisplayBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>210</y>
      <width>251</width>
      <height>211</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="infolabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Lists of companies</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addCompanies">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add companies</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>180</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Profit Companies</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>180</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Non-Profit Companies</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>531</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
